docs/reference/troubleshooting/corruption-issues.asciidoc

```diff
@@ -50,8 +57,7 @@ system previously confirmed to {es} that this file was durably synced to disk.
 On Linux this means that the `fsync()` system call returned successfully. {es}
 sometimes reports that an index is corrupt because a file needed for recovery
 is missing, or it exists but has been truncated or is missing its footer. This
-indicates that your storage system acknowledges durable writes incorrectly or
-that some external process has modified the data {es} previously wrote to disk.
+may indicate that your storage system acknowledges durable writes incorrectly.
```

Comment:
I do not think this change is related to the title. Was it intentional?

Reply:
Yeah it was intentional, I meant to fix this in the last change to this file but missed it.

Reply:
Why would you like to remove the mention of another process? My understanding is that it is still a possibility. Is it mentioned anywhere else?